### PR TITLE
add `fa_IR` 

### PR DESCRIPTION
**Pull request by hamidb80**
Hey, I added `fa_IR` support for few modules

### CODE REVIEW

**Comment by hamidb80** [comment]
I think it's better to store datasets in separated files btw.

**Comment by jiro4989** [comment]
Thank you.

**Comment by jiro4989** [comment]
Please implements `src/faker/provider/internet/internet_fa_IR.nim`.
Because, `proc email*(f: Faker): string` ([here](https://github.com/hamidb80/faker/blob/926693ca950fb90e7b7d8ebac6a87f6b6d778beb/src/faker/provider/internet.nim#L65-L74)) returns illegal email.

ex:

```
بهزاد@example.org
الهام@yahoo.co.jp
مسعود@example.com
```

see: [Internet Message Format > Address Specification - RFC5322](https://datatracker.ietf.org/doc/html/rfc5322#section-3.4)

**Review by jiro4989 — CHANGES_REQUESTED** [review]
Please add test code here

https://github.com/hamidb80/faker/blob/master/tests/test1.nim#L12

```nim
let locales = ["en_US", "ja_JP", "fa_IR"]
```

**Review by jiro4989 — CHANGES_REQUESTED** [review]
Please implements `src/faker/provider/internet/internet_fa_IR.nim`.
Because, `proc email*(f: Faker): string` ([here](https://github.com/hamidb80/faker/blob/926693ca950fb90e7b7d8ebac6a87f6b6d778beb/src/faker/provider/internet.nim#L65-L74)) returns illegal email.

ex:

```
بهزاد@example.org
الهام@yahoo.co.jp
مسعود@example.com
```

see: [Internet Message Format > Address Specification - RFC5322](https://datatracker.ietf.org/doc/html/rfc5322#section-3.4)

**Comment by hamidb80** [comment]
I think it's OK now

File: src/faker/provider/internet/internet_fa_IR.nim [hunk 1]
```diff
@@ -0,0 +1,22 @@
+include interfaces
+import strutils
+import ../person/person_fa_IR
+
+proc pickFinglish(f: Faker): string =
+  f.rand.sample(finglish) & f.rand.sample(finglish) & (
+    if rand(1..5) < 5: $rand(1..9999)
+    else: ""
```
Comment (jiro4989) [inline src/faker/provider/internet/internet_fa_IR.nim:8]: hmm...?
Compile error was occured in Nim 1.0.x, and 1.2.x.

Please check this [CI Log](https://github.com/jiro4989/faker/runs/4782151899?check_suite_focus=true#step:4:10) and fix it.

**Review by jiro4989 — CHANGES_REQUESTED** [review]

**Review by jiro4989 — APPROVED** [review]
LGTM 👍 

LGTM

**Pull request merged**